index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
 
   contentFor: function(type) {
     if (type === 'vendor-suffix') {
-      return 'if (typeof(document) !== "undefined") {\n' + fs.readFileSync(path.join(__dirname, 'vendor/jquery-waypoints/waypoints.js')) + '\n}';
+      return 'if (typeof(document) !== "undefined") {\n' + fs.readFileSync(path.join(__dirname, '../../bower_components/jquery-waypoints/waypoints.js')) + '\n}';
```

Comment:
hmmmm, same error.... It's trying to find the waypoint.js file in the bower_components of the addon's node_modules folder instead of the application's bower-componets folder.